calendar_af/wizard/create_local_occasion.py

```diff
@@ -220,6 +220,8 @@ def _action_create_occasions(self):
                 repeat_list.append(3)
             if self.repeat_fri:
                 repeat_list.append(4)
+            if not (self.repeat_mon or self.repeat_tue or self.repeat_wed or self.repeat_thu or self.repeat_fri):
```

Comment:
easier to write if not repeat_list: ?

Reply:
Maybe think if its possible that people work on sat/sun?

Reply:
Changed it to repeat_list, we don't allow the creation of occasions on sat/sun at the moment.